nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -551,6 +551,8 @@ The module update takes care of the new config syntax and the data itself (user
 
 - The argument `vendorSha256` of `buildGoModule` is deprecated. Use `vendorHash` instead. ([\#259999](https://github.com/NixOS/nixpkgs/pull/259999))
 
+- TeX Live environments can now be built with the new `texlive.withPackages` and creating custom TeX packages has been simplified.
```

Comment:
```suggestion
- TeX Live environments can now be built with the new `texlive.withPackages`. Procedure for creating custom TeX packages has been changed, see the [Nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#sec-language-texlive-custom-packages) for more details.
```

Reply:
Should we also say `Existing custom packages are still supported.`?

Reply:
It's probably not. There are no tests, and I assume, you plan to deprecate/remove it.